### PR TITLE
Lint, debug and runs using make

### PR DESCRIPTION
**Pull request by lior-blox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,17 +78,16 @@ $ ./bin/ssv --help
...
-
 ```bash 
-# Build nodes
-$ docker-compose build
+# Run 3 nodes (prod mode)
+$ make docker
 
-# Run nodes
-$ docker-compose up -d
+# Run 3 nodes (debug & hot reload mode)
```
Comment (vinipx) [inline README.md:89]: minor: "live reload" instead of "hot reload"? they are indeed different modes.

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,68 +1,106 @@
...
-      - "--beacon-node-addr=${BEACON_NODE_ADDR}"
-      - "--network=${NETWORK}"
-      - "--val=${CONSENSUS_TYPE}"
+    <<: *default-base
+    container_name: ssv-node-1
     environment:
       DOCKER_COMPOSE: "true"
+      NODE_ID: 1
+      SSV_PRIVATE_KEY: ${SSV_NODE_1}
       PUBKEY_NODE_2: "98090f5ace3818b27b6ea2d38524556ec8c39d568a1595adb00964415eae62998c12ef015eda17f8d590f22b1778ec02"
```
Comment (vinipx) [inline docker-compose.yaml]: **Out-of-Scope but as a suggestion:** maybe we could parameterized the keys instead of hard-coding them here?

**Review by vinipx — COMMENTED** [review]
Cool! added some minors comments

File: docker-compose.yaml (hunk 2 above)
Comment (lior-blox) [reply docker-compose.yaml]: already did

**Comment by vinipx** [comment]
lgtm 👍

**Comment by roman-blox** [comment]
CI pipeline config should be updated as well. Good work btw!

**Pull request merged**